cmd/dep/testdata/harness_tests/remove/specific/case2/testcase.json

```diff
@@ -2,6 +2,7 @@
   "commands": [
     ["remove", "github.com/not/used"]
   ],
+  "error-expected": "",
```

Comment:
👍  I've been seeing this pop up when I run tests locally. Any idea why the build didn't break and force us to update the testdata when it was first introduced?